.github/workflows/post-release.yml

```
@@ -62,12 +62,20 @@ jobs:
       - name: Update GitHub action config
         run: make assets/github-action-config.json
 
+      - name: Update reference files
+        run: |
+          rm .golangci.reference.yml
+          cp .golangci.next.reference.yml .golangci.reference.yml
```

Was `rm + cp` intentionally used instead of just `mv`?"

```suggestion
          mv .golangci.next.reference.yml .golangci.reference.yml
```

---

I think it's because I have a zsh (default?) `cp` alias: 
```console
$ alias        
cp='cp -i'
```